apps/dav/src/components/ExampleContactSettings.vue

```diff
@@ -6,9 +6,9 @@
 <template>
 	<div class="example-contact-settings">
 		<NcCheckboxRadioSwitch
-			:checked="enableDefaultContact"
+			:modelValue="enableDefaultContact"
```

Comment:
I wonder why the linter is not not red as props should be kebab-case


```suggestion
			:model-value="enableDefaultContact"
```